DEBUG   2018-07-13T14:17:41.759Z - GIVEN              | >> Apply Given 
INFO    2018-07-13T14:17:41.765Z - COMMAND            | npmInstallLatestVersion(testing-projects/empty-app) 
INFO    2018-07-13T14:17:41.764Z - GIVEN              | >>> Given testing application: testing-projects/empty-app 
SILLY   2018-07-13T14:17:41.765Z - GIVEN              | GivenTestingApp:execute -> npmInstallLatestVersion(testing-projects/empty-app) 
SILLY   2018-07-13T14:17:42.215Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] 
SILLY   2018-07-13T14:17:46.532Z - PROCESS STDOUT/STD | 

> get-cursor-position@1.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position
> node-gyp rebuild

make : on entre dans le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »
  CXX(target) Release/obj.target/pos/src/pos.o
  SOLINK_MODULE(target) Release/obj.target/pos.node
  COPY Release/pos.node
make : on quitte le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »

> bufferutil@3.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/bufferutil
> prebuild-install || node-gyp rebuild


> utf-8-validate@4.0.2 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/utf-8-validate
> prebuild-install || node-gyp rebuild

+ @zetapush/cli@0.30.0-alpha.743bc6df
added 208 packages in 3.938s 
SILLY   2018-07-13T14:17:46.532Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] ->  
SILLY   2018-07-13T14:17:46.533Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] 
SILLY   2018-07-13T14:17:48.288Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] ->  
SILLY   2018-07-13T14:17:48.290Z - COMMAND            | symlinkLocalDependencies(testing-projects/empty-app) 
WARN    2018-07-13T14:17:46.533Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^ 
SILLY   2018-07-13T14:17:48.289Z - PROCESS STDOUT/STD | 
+ @zetapush/platform@0.30.0-alpha.743bc6df
updated 1 package in 1.397s 
WARN    2018-07-13T14:17:48.289Z - PROCESS STDOUT/STD | 
 
SILLY   2018-07-13T14:17:48.586Z - GIVEN              | GivenTestingApp:execute -> setAccountToZetarc(testing-projects/empty-app, {"developerLogin":"accountnotvalidated@zetapush.com","developerPassword":"password"}) 
DEBUG   2018-07-13T14:17:48.590Z - COMMAND            | readZetarc(testing-projects/empty-app) 
DEBUG   2018-07-13T14:17:48.591Z - COMMAND            | readZetarc(testing-projects/empty-app) ->  
INFO    2018-07-13T14:17:48.592Z - USER ACTION        | >>> User action: zeta push from console 
ERROR   2018-07-13T14:17:50.958Z - PROCESS STDOUT/STD | 

> empty-project@0.0.1 deploy /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app
> zeta push "-vvv"

[LOG] Load and merge configuration with this priority order:
      1) command line arguments
      2) environment variables
      3) from file defined in ./.zetarc  ‌
[TRACE] Try to load conf from cli provess.argv  ‌
[TRACE] Try to load conf from process.env  ‌
[TRACE] Try to load conf from filesystem [ '.' ] ‌
[TRACE] Configuration loaded from file [ '.',
  { config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       developerLogin: 'accountnotvalidated@zetapush.com',
       developerPassword: 'password' },
    filepath: '/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/.zetarc' } ] ‌
[TRACE] Using default values  ‌
[TRACE] merged config [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'accountnotvalidated@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[TRACE] config is valid [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'accountnotvalidated@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/auth/whoami', undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/cluster/list',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/cluster/list',
  IncomingMessage {
    _readableState: 
     ReadableState {
       objectMode: false,
       highWaterMark: 16384,
       buffer: [Object],
       length: 0,
       pipes: null,
       pipesCount: 0,
       flowing: true,
       ended: true,
       endEmitted: true,
       reading: false,
       sync: false,
       needReadable: false,
       emittedReadable: false,
       readableListening: false,
       resumeScheduled: false,
       destroyed: false,
       defaultEncoding: 'utf8',
       awaitDrain: 0,
       readingMore: false,
       decoder: null,
       encoding: null },
    readable: false,
    domain: null,
    _events: 
     { end: [Array],
       close: [Array],
       data: [Function],
       error: [Function] },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 219,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 168,
       [Symbol(bytesRead)]: 358 },
    connection: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 219,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 168,
       [Symbol(bytesRead)]: 358 },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: 
     { 'x-content-type-options': 'nosniff',
       'x-xss-protection': '1; mode=block',
       'cache-control': 'no-cache, no-store, max-age=0, must-revalidate',
       pragma: 'no-cache',
       expires: '0',
       'x-frame-options': 'DENY',
       connection: 'close',
       server: 'Jetty(9.2.8.v20150217)' },
    rawHeaders: 
     [ 'X-Content-Type-Options',
       'nosniff',
       'X-XSS-Protection',
       '1; mode=block',
       'Cache-Control',
       'no-cache, no-store, max-age=0, must-revalidate',
       'Pragma',
       'no-cache',
       'Expires',
       '0',
       'X-Frame-Options',
       'DENY',
       'Connection',
       'close',
       'Server',
       'Jetty(9.2.8.v20150217)' ],
    trailers: {},
    rawTrailers: [],
    upgrade: false,
    url: '',
    method: null,
    statusCode: 403,
    statusMessage: 'Forbidden',
    client: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 219,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 168,
       [Symbol(bytesRead)]: 358 },
    _consuming: true,
    _dumped: false,
    req: 
     ClientRequest {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       output: [],
       outputEncodings: [],
       outputCallbacks: [],
       outputSize: 0,
       writable: true,
       _last: true,
       upgrading: false,
       chunkedEncoding: false,
       shouldKeepAlive: false,
       useChunkedEncodingByDefault: false,
       sendDate: false,
       _removedConnection: false,
       _removedContLen: false,
       _removedTE: false,
       _contentLength: 0,
       _hasBody: true,
       _trailer: '',
       finished: true,
       _headerSent: true,
       socket: [Object],
       connection: [Object],
       _header: 'GET /zbo/orga/cluster/list HTTP/1.1\r\nContent-Type: application/json;charset=UTF-8\r\nX-Authorization: {"username":"accountnotvalidated@zetapush.com","password":"password"}\r\nhost: celtia.zetapush.com\r\nConnection: close\r\n\r\n',
       _onPendingData: [Function: noopPendingOutput],
       agent: [Object],
       socketPath: undefined,
       timeout: undefined,
       method: 'GET',
       path: '/zbo/orga/cluster/list',
       _ended: true,
       res: [Circular],
       aborted: undefined,
       timeoutCb: null,
       upgradeOrConnect: false,
       parser: null,
       maxHeadersCount: null,
       [Symbol(outHeadersKey)]: [Object] },
    request: 
     Request {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       headers: [Object],
       method: 'GET',
       callback: [Function],
       readable: true,
       writable: true,
       explicitMethod: true,
       _qs: [Object],
       _auth: [Object],
       _oauth: [Object],
       _multipart: [Object],
       _redirect: [Object],
       _tunnel: [Object],
       setHeader: [Function],
       hasHeader: [Function],
       getHeader: [Function],
       removeHeader: [Function],
       localAddress: undefined,
       pool: {},
       dests: [],
       __isRequestRequest: true,
       _callback: [Function],
       uri: [Object],
       proxy: [Object],
       tunnel: true,
       agent: [Object],
       setHost: true,
       originalCookieHeader: undefined,
       _disableCookies: true,
       _jar: undefined,
       port: 443,
       host: 'celtia.zetapush.com',
       path: '/zbo/orga/cluster/list',
       httpModule: [Object],
       _started: true,
       href: 'https://celtia.zetapush.com/zbo/orga/cluster/list',
       req: [Object],
       ntick: true,
       response: [Circular],
       originalHost: 'celtia.zetapush.com',
       originalHostHeaderName: 'host',
       responseContent: [Circular],
       _destdata: true,
       _ended: true,
       _callbackCalled: true },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    read: [Function],
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}' } ] ‌
[TRACE] not config issue  ‌
[TRACE] not a network error  ‌
[TRACE] access denied [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}',
    request: 
     { body: undefined,
       headers: [Object],
       method: 'GET',
       url: 'https://celtia.zetapush.com:/zbo/orga/cluster/list' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotvalidated@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
[TRACE] account not validated  ‌
[TRACE] Analyze done => errorCode=ACCOUNT-05. Displaying help for this code  ‌
[TRACE] Display help message for ACCOUNT-05  ‌
[TRACE] Cache expired=false  ‌
[TRACE] Use cached troubleshoot for ACCOUNT-05  ‌
(?) 
You have created a ZetaPush account but the account is not confirmed.

When you created your account, we sent you an email with a confirmation link.
You must click the confirmation link in order to validate your account and be able to use ZetaPush services.
This confirmation link is valid for 7 days.
  ‌
 
INFO    2018-07-13T14:17:48.593Z - COMMAND            | zetaPush(testing-projects/empty-app) -> [npm run deploy -- -vvv] 
ERROR   2018-07-13T14:17:50.959Z - COMMAND            | zetaPush(testing-projects/empty-app) -> [npm run deploy -- -vvv] {"exitCode":55}
INFO    2018-07-13T14:17:50.960Z - USER ACTION        | >>> User action DONE: zeta push from console 
ERROR   2018-07-13T14:17:50.959Z - PROCESS STDOUT/STD | 
[ERROR] Push failed [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}',
    request: 
     { body: undefined,
       headers: [Object],
       method: 'GET',
       url: 'https://celtia.zetapush.com:/zbo/orga/cluster/list' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotvalidated@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
npm ERR! code ELIFECYCLE
npm ERR! errno 55
npm ERR! empty-project@0.0.1 deploy: `zeta push "-vvv"`
npm ERR! Exit status 55
npm ERR! 
npm ERR! Failed at the empty-project@0.0.1 deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/aurelien/.npm/_logs/2018-07-13T14_17_50_952Z-debug.log
 
DEBUG   2018-07-13T14:17:50.970Z - GIVEN              | >> Apply Given 
INFO    2018-07-13T14:17:50.972Z - COMMAND            | npmInstallLatestVersion(testing-projects/empty-app) 
INFO    2018-07-13T14:17:50.971Z - GIVEN              | >>> Given testing application: testing-projects/empty-app 
SILLY   2018-07-13T14:17:50.971Z - GIVEN              | GivenTestingApp:execute -> npmInstallLatestVersion(testing-projects/empty-app) 
SILLY   2018-07-13T14:17:51.386Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] 
SILLY   2018-07-13T14:17:56.150Z - PROCESS STDOUT/STD | 

> get-cursor-position@1.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position
> node-gyp rebuild

make : on entre dans le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »
  CXX(target) Release/obj.target/pos/src/pos.o
  SOLINK_MODULE(target) Release/obj.target/pos.node
  COPY Release/pos.node
make : on quitte le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »

> bufferutil@3.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/bufferutil
> prebuild-install || node-gyp rebuild


> utf-8-validate@4.0.2 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/utf-8-validate
> prebuild-install || node-gyp rebuild

+ @zetapush/cli@0.30.0-alpha.743bc6df
added 208 packages in 4.391s 
SILLY   2018-07-13T14:17:56.149Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] ->  
SILLY   2018-07-13T14:17:56.151Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] 
SILLY   2018-07-13T14:17:57.994Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] ->  
SILLY   2018-07-13T14:17:57.995Z - COMMAND            | symlinkLocalDependencies(testing-projects/empty-app) 
WARN    2018-07-13T14:17:56.151Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^ 
SILLY   2018-07-13T14:17:57.995Z - PROCESS STDOUT/STD | 
+ @zetapush/platform@0.30.0-alpha.743bc6df
updated 1 package in 1.439s 
WARN    2018-07-13T14:17:57.995Z - PROCESS STDOUT/STD | 
 
SILLY   2018-07-13T14:17:58.289Z - GIVEN              | GivenTestingApp:execute -> setAccountToZetarc(testing-projects/empty-app, {"developerLogin":"accountnotvalidated@zetapush.com","developerPassword":"password"}) 
DEBUG   2018-07-13T14:17:58.291Z - COMMAND            | readZetarc(testing-projects/empty-app) 
DEBUG   2018-07-13T14:17:58.292Z - COMMAND            | readZetarc(testing-projects/empty-app) ->  
INFO    2018-07-13T14:17:58.292Z - USER ACTION        | >>> User action: zeta run from console 
ERROR   2018-07-13T14:17:59.910Z - PROCESS STDOUT/STD | 

> empty-project@0.0.1 start /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app
> zeta run "-vvv"

[LOG] Load and merge configuration with this priority order:
      1) command line arguments
      2) environment variables
      3) from file defined in ./.zetarc  ‌
[TRACE] Try to load conf from cli provess.argv  ‌
[TRACE] Try to load conf from process.env  ‌
[TRACE] Try to load conf from filesystem [ '.' ] ‌
[TRACE] Configuration loaded from file [ '.',
  { config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       developerLogin: 'accountnotvalidated@zetapush.com',
       developerPassword: 'password' },
    filepath: '/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/.zetarc' } ] ‌
[TRACE] Using default values  ‌
[TRACE] merged config [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'accountnotvalidated@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[TRACE] config is valid [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'accountnotvalidated@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/auth/whoami', undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/cluster/list',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/auth/whoami',
  IncomingMessage {
    _readableState: 
     ReadableState {
       objectMode: false,
       highWaterMark: 16384,
       buffer: [Object],
       length: 0,
       pipes: null,
       pipesCount: 0,
       flowing: true,
       ended: true,
       endEmitted: true,
       reading: false,
       sync: false,
       needReadable: false,
       emittedReadable: false,
       readableListening: false,
       resumeScheduled: false,
       destroyed: false,
       defaultEncoding: 'utf8',
       awaitDrain: 0,
       readingMore: false,
       decoder: null,
       encoding: null },
    readable: false,
    domain: null,
    _events: 
     { end: [Array],
       close: [Array],
       data: [Function],
       error: [Function] },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 213,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 159,
       [Symbol(bytesRead)]: 358 },
    connection: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 213,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 159,
       [Symbol(bytesRead)]: 358 },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: 
     { 'x-content-type-options': 'nosniff',
       'x-xss-protection': '1; mode=block',
       'cache-control': 'no-cache, no-store, max-age=0, must-revalidate',
       pragma: 'no-cache',
       expires: '0',
       'x-frame-options': 'DENY',
       connection: 'close',
       server: 'Jetty(9.2.8.v20150217)' },
    rawHeaders: 
     [ 'X-Content-Type-Options',
       'nosniff',
       'X-XSS-Protection',
       '1; mode=block',
       'Cache-Control',
       'no-cache, no-store, max-age=0, must-revalidate',
       'Pragma',
       'no-cache',
       'Expires',
       '0',
       'X-Frame-Options',
       'DENY',
       'Connection',
       'close',
       'Server',
       'Jetty(9.2.8.v20150217)' ],
    trailers: {},
    rawTrailers: [],
    upgrade: false,
    url: '',
    method: null,
    statusCode: 403,
    statusMessage: 'Forbidden',
    client: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 213,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 159,
       [Symbol(bytesRead)]: 358 },
    _consuming: true,
    _dumped: false,
    req: 
     ClientRequest {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       output: [],
       outputEncodings: [],
       outputCallbacks: [],
       outputSize: 0,
       writable: true,
       _last: true,
       upgrading: false,
       chunkedEncoding: false,
       shouldKeepAlive: false,
       useChunkedEncodingByDefault: false,
       sendDate: false,
       _removedConnection: false,
       _removedContLen: false,
       _removedTE: false,
       _contentLength: 0,
       _hasBody: true,
       _trailer: '',
       finished: true,
       _headerSent: true,
       socket: [Object],
       connection: [Object],
       _header: 'GET /zbo/auth/whoami HTTP/1.1\r\nContent-Type: application/json;charset=UTF-8\r\nX-Authorization: {"username":"accountnotvalidated@zetapush.com","password":"password"}\r\nhost: celtia.zetapush.com\r\nConnection: close\r\n\r\n',
       _onPendingData: [Function: noopPendingOutput],
       agent: [Object],
       socketPath: undefined,
       timeout: undefined,
       method: 'GET',
       path: '/zbo/auth/whoami',
       _ended: true,
       res: [Circular],
       aborted: undefined,
       timeoutCb: null,
       upgradeOrConnect: false,
       parser: null,
       maxHeadersCount: null,
       [Symbol(outHeadersKey)]: [Object] },
    request: 
     Request {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       headers: [Object],
       method: 'GET',
       callback: [Function],
       readable: true,
       writable: true,
       explicitMethod: true,
       _qs: [Object],
       _auth: [Object],
       _oauth: [Object],
       _multipart: [Object],
       _redirect: [Object],
       _tunnel: [Object],
       setHeader: [Function],
       hasHeader: [Function],
       getHeader: [Function],
       removeHeader: [Function],
       localAddress: undefined,
       pool: {},
       dests: [],
       __isRequestRequest: true,
       _callback: [Function],
       uri: [Object],
       proxy: [Object],
       tunnel: true,
       agent: [Object],
       setHost: true,
       originalCookieHeader: undefined,
       _disableCookies: true,
       _jar: undefined,
       port: 443,
       host: 'celtia.zetapush.com',
       path: '/zbo/auth/whoami',
       httpModule: [Object],
       _started: true,
       href: 'https://celtia.zetapush.com/zbo/auth/whoami',
       req: [Object],
       ntick: true,
       response: [Circular],
       originalHost: 'celtia.zetapush.com',
       originalHostHeaderName: 'host',
       responseContent: [Circular],
       _destdata: true,
       _ended: true,
       _callbackCalled: true },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    read: [Function],
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}' } ] ‌
[TRACE] not config issue  ‌
[TRACE] not a network error  ‌
[TRACE] access denied [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}',
    request: 
     { body: undefined,
       headers: [Object],
       method: 'GET',
       url: 'https://celtia.zetapush.com:/zbo/auth/whoami' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotvalidated@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
[TRACE] account not validated  ‌
[TRACE] Analyze done => errorCode=ACCOUNT-05. Displaying help for this code  ‌
[TRACE] Display help message for ACCOUNT-05  ‌
[TRACE] Cache expired=false  ‌
[TRACE] Use cached troubleshoot for ACCOUNT-05  ‌
(?) 
You have created a ZetaPush account but the account is not confirmed.

When you created your account, we sent you an email with a confirmation link.
You must click the confirmation link in order to validate your account and be able to use ZetaPush services.
This confirmation link is valid for 7 days.
  ‌
 
INFO    2018-07-13T14:17:58.293Z - COMMAND            | zetaRun(testing-projects/empty-app) -> [npm run start -- -vvv] 
ERROR   2018-07-13T14:17:59.912Z - COMMAND            | zetaRun(testing-projects/empty-app) -> [npm run start -- -vvv] {"exitCode":55}
INFO    2018-07-13T14:17:59.912Z - USER ACTION        | >>> User action DONE: zeta run from console 
ERROR   2018-07-13T14:17:59.911Z - PROCESS STDOUT/STD | 
[ERROR] Run failed [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden","context":{"reason":"ACCOUNT_DISABLED"}}',
    request: 
     { body: undefined,
       headers: [Object],
       method: 'GET',
       url: 'https://celtia.zetapush.com:/zbo/auth/whoami' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotvalidated@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
npm ERR! code ELIFECYCLE
npm ERR! errno 55
npm ERR! empty-project@0.0.1 start: `zeta run "-vvv"`
npm ERR! Exit status 55
npm ERR! 
npm ERR! Failed at the empty-project@0.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/aurelien/.npm/_logs/2018-07-13T14_17_59_906Z-debug.log
 
DEBUG   2018-07-13T14:17:59.941Z - GIVEN              | >> Apply Given 
INFO    2018-07-13T14:17:59.942Z - COMMAND            | npmInstallLatestVersion(testing-projects/empty-app) 
INFO    2018-07-13T14:17:59.942Z - GIVEN              | >>> Given testing application: testing-projects/empty-app 
SILLY   2018-07-13T14:17:59.942Z - GIVEN              | GivenTestingApp:execute -> npmInstallLatestVersion(testing-projects/empty-app) 
SILLY   2018-07-13T14:18:00.348Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] 
SILLY   2018-07-13T14:18:05.213Z - PROCESS STDOUT/STD | 

> get-cursor-position@1.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position
> node-gyp rebuild

make : on entre dans le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »
  CXX(target) Release/obj.target/pos/src/pos.o
  SOLINK_MODULE(target) Release/obj.target/pos.node
  COPY Release/pos.node
make : on quitte le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »

> bufferutil@3.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/bufferutil
> prebuild-install || node-gyp rebuild


> utf-8-validate@4.0.2 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/utf-8-validate
> prebuild-install || node-gyp rebuild

+ @zetapush/cli@0.30.0-alpha.743bc6df
added 208 packages in 4.492s 
SILLY   2018-07-13T14:18:05.212Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] ->  
SILLY   2018-07-13T14:18:05.214Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] 
SILLY   2018-07-13T14:18:07.119Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] ->  
SILLY   2018-07-13T14:18:07.120Z - COMMAND            | symlinkLocalDependencies(testing-projects/empty-app) 
WARN    2018-07-13T14:18:05.213Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^ 
SILLY   2018-07-13T14:18:07.120Z - PROCESS STDOUT/STD | 
+ @zetapush/platform@0.30.0-alpha.743bc6df
updated 1 package in 1.5s 
WARN    2018-07-13T14:18:07.120Z - PROCESS STDOUT/STD | 
 
SILLY   2018-07-13T14:18:07.428Z - GIVEN              | GivenTestingApp:execute -> setAccountToZetarc(testing-projects/empty-app, {"developerLogin":"accountnotexists@zetapush.com","developerPassword":"password"}) 
DEBUG   2018-07-13T14:18:07.430Z - COMMAND            | readZetarc(testing-projects/empty-app) 
DEBUG   2018-07-13T14:18:07.431Z - COMMAND            | readZetarc(testing-projects/empty-app) ->  
INFO    2018-07-13T14:18:07.431Z - USER ACTION        | >>> User action: zeta push from console 
ERROR   2018-07-13T14:18:09.095Z - PROCESS STDOUT/STD | 

> empty-project@0.0.1 deploy /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app
> zeta push "-vvv"

[LOG] Load and merge configuration with this priority order:
      1) command line arguments
      2) environment variables
      3) from file defined in ./.zetarc  ‌
[TRACE] Try to load conf from cli provess.argv  ‌
[TRACE] Try to load conf from process.env  ‌
[TRACE] Try to load conf from filesystem [ '.' ] ‌
[TRACE] Configuration loaded from file [ '.',
  { config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       developerLogin: 'accountnotexists@zetapush.com',
       developerPassword: 'password' },
    filepath: '/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/.zetarc' } ] ‌
[TRACE] Using default values  ‌
[TRACE] merged config [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'accountnotexists@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[TRACE] config is valid [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'accountnotexists@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/auth/whoami', undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/cluster/list',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/cluster/list',
  IncomingMessage {
    _readableState: 
     ReadableState {
       objectMode: false,
       highWaterMark: 16384,
       buffer: [Object],
       length: 0,
       pipes: null,
       pipesCount: 0,
       flowing: true,
       ended: true,
       endEmitted: true,
       reading: false,
       sync: false,
       needReadable: false,
       emittedReadable: false,
       readableListening: false,
       resumeScheduled: false,
       destroyed: false,
       defaultEncoding: 'utf8',
       awaitDrain: 0,
       readingMore: false,
       decoder: null,
       encoding: null },
    readable: false,
    domain: null,
    _events: 
     { end: [Array],
       close: [Array],
       data: [Function],
       error: [Function] },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 216,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 159,
       [Symbol(bytesRead)]: 318 },
    connection: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 216,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 159,
       [Symbol(bytesRead)]: 318 },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: 
     { 'x-content-type-options': 'nosniff',
       'x-xss-protection': '1; mode=block',
       'cache-control': 'no-cache, no-store, max-age=0, must-revalidate',
       pragma: 'no-cache',
       expires: '0',
       'x-frame-options': 'DENY',
       connection: 'close',
       server: 'Jetty(9.2.8.v20150217)' },
    rawHeaders: 
     [ 'X-Content-Type-Options',
       'nosniff',
       'X-XSS-Protection',
       '1; mode=block',
       'Cache-Control',
       'no-cache, no-store, max-age=0, must-revalidate',
       'Pragma',
       'no-cache',
       'Expires',
       '0',
       'X-Frame-Options',
       'DENY',
       'Connection',
       'close',
       'Server',
       'Jetty(9.2.8.v20150217)' ],
    trailers: {},
    rawTrailers: [],
    upgrade: false,
    url: '',
    method: null,
    statusCode: 403,
    statusMessage: 'Forbidden',
    client: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 216,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 159,
       [Symbol(bytesRead)]: 318 },
    _consuming: true,
    _dumped: false,
    req: 
     ClientRequest {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       output: [],
       outputEncodings: [],
       outputCallbacks: [],
       outputSize: 0,
       writable: true,
       _last: true,
       upgrading: false,
       chunkedEncoding: false,
       shouldKeepAlive: false,
       useChunkedEncodingByDefault: false,
       sendDate: false,
       _removedConnection: false,
       _removedContLen: false,
       _removedTE: false,
       _contentLength: 0,
       _hasBody: true,
       _trailer: '',
       finished: true,
       _headerSent: true,
       socket: [Object],
       connection: [Object],
       _header: 'GET /zbo/orga/cluster/list HTTP/1.1\r\nContent-Type: application/json;charset=UTF-8\r\nX-Authorization: {"username":"accountnotexists@zetapush.com","password":"password"}\r\nhost: celtia.zetapush.com\r\nConnection: close\r\n\r\n',
       _onPendingData: [Function: noopPendingOutput],
       agent: [Object],
       socketPath: undefined,
       timeout: undefined,
       method: 'GET',
       path: '/zbo/orga/cluster/list',
       _ended: true,
       res: [Circular],
       aborted: undefined,
       timeoutCb: null,
       upgradeOrConnect: false,
       parser: null,
       maxHeadersCount: null,
       [Symbol(outHeadersKey)]: [Object] },
    request: 
     Request {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       headers: [Object],
       method: 'GET',
       callback: [Function],
       readable: true,
       writable: true,
       explicitMethod: true,
       _qs: [Object],
       _auth: [Object],
       _oauth: [Object],
       _multipart: [Object],
       _redirect: [Object],
       _tunnel: [Object],
       setHeader: [Function],
       hasHeader: [Function],
       getHeader: [Function],
       removeHeader: [Function],
       localAddress: undefined,
       pool: {},
       dests: [],
       __isRequestRequest: true,
       _callback: [Function],
       uri: [Object],
       proxy: [Object],
       tunnel: true,
       agent: [Object],
       setHost: true,
       originalCookieHeader: undefined,
       _disableCookies: true,
       _jar: undefined,
       port: 443,
       host: 'celtia.zetapush.com',
       path: '/zbo/orga/cluster/list',
       httpModule: [Object],
       _started: true,
       href: 'https://celtia.zetapush.com/zbo/orga/cluster/list',
       req: [Object],
       ntick: true,
       response: [Circular],
       originalHost: 'celtia.zetapush.com',
       originalHostHeaderName: 'host',
       responseContent: [Circular],
       _destdata: true,
       _ended: true,
       _callbackCalled: true },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    read: [Function],
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' } ] ‌
[TRACE] not config issue  ‌
[TRACE] not a network error  ‌
[TRACE] access denied [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}',
    request: 
     { body: undefined,
       headers: [Object],
       method: 'GET',
       url: 'https://celtia.zetapush.com:/zbo/orga/cluster/list' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotexists@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
[LOG] POST [ 'https://celtia.zetapush.com:/zbo//auth/login',
  '{"username":"accountnotexists@zetapush.com","password":"password"}' ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/auth/whoami',
  IncomingMessage {
    _readableState: 
     ReadableState {
       objectMode: false,
       highWaterMark: 16384,
       buffer: [Object],
       length: 0,
       pipes: null,
       pipesCount: 0,
       flowing: true,
       ended: true,
       endEmitted: true,
       reading: false,
       sync: false,
       needReadable: false,
       emittedReadable: false,
       readableListening: false,
       resumeScheduled: false,
       destroyed: false,
       defaultEncoding: 'utf8',
       awaitDrain: 0,
       readingMore: false,
       decoder: null,
       encoding: null },
    readable: false,
    domain: null,
    _events: 
     { end: [Array],
       close: [Array],
       data: [Function],
       error: [Function] },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 210,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 168,
       [Symbol(bytesRead)]: 318 },
    connection: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 210,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 168,
       [Symbol(bytesRead)]: 318 },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: 
     { 'x-content-type-options': 'nosniff',
       'x-xss-protection': '1; mode=block',
       'cache-control': 'no-cache, no-store, max-age=0, must-revalidate',
       pragma: 'no-cache',
       expires: '0',
       'x-frame-options': 'DENY',
       connection: 'close',
       server: 'Jetty(9.2.8.v20150217)' },
    rawHeaders: 
     [ 'X-Content-Type-Options',
       'nosniff',
       'X-XSS-Protection',
       '1; mode=block',
       'Cache-Control',
       'no-cache, no-store, max-age=0, must-revalidate',
       'Pragma',
       'no-cache',
       'Expires',
       '0',
       'X-Frame-Options',
       'DENY',
       'Connection',
       'close',
       'Server',
       'Jetty(9.2.8.v20150217)' ],
    trailers: {},
    rawTrailers: [],
    upgrade: false,
    url: '',
    method: null,
    statusCode: 403,
    statusMessage: 'Forbidden',
    client: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 210,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 168,
       [Symbol(bytesRead)]: 318 },
    _consuming: true,
    _dumped: false,
    req: 
     ClientRequest {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       output: [],
       outputEncodings: [],
       outputCallbacks: [],
       outputSize: 0,
       writable: true,
       _last: true,
       upgrading: false,
       chunkedEncoding: false,
       shouldKeepAlive: false,
       useChunkedEncodingByDefault: false,
       sendDate: false,
       _removedConnection: false,
       _removedContLen: false,
       _removedTE: false,
       _contentLength: 0,
       _hasBody: true,
       _trailer: '',
       finished: true,
       _headerSent: true,
       socket: [Object],
       connection: [Object],
       _header: 'GET /zbo/auth/whoami HTTP/1.1\r\nContent-Type: application/json;charset=UTF-8\r\nX-Authorization: {"username":"accountnotexists@zetapush.com","password":"password"}\r\nhost: celtia.zetapush.com\r\nConnection: close\r\n\r\n',
       _onPendingData: [Function: noopPendingOutput],
       agent: [Object],
       socketPath: undefined,
       timeout: undefined,
       method: 'GET',
       path: '/zbo/auth/whoami',
       _ended: true,
       res: [Circular],
       aborted: undefined,
       timeoutCb: null,
       upgradeOrConnect: false,
       parser: null,
       maxHeadersCount: null,
       [Symbol(outHeadersKey)]: [Object] },
    request: 
     Request {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       headers: [Object],
       method: 'GET',
       callback: [Function],
       readable: true,
       writable: true,
       explicitMethod: true,
       _qs: [Object],
       _auth: [Object],
       _oauth: [Object],
       _multipart: [Object],
       _redirect: [Object],
       _tunnel: [Object],
       setHeader: [Function],
       hasHeader: [Function],
       getHeader: [Function],
       removeHeader: [Function],
       localAddress: undefined,
       pool: {},
       dests: [],
       __isRequestRequest: true,
       _callback: [Function],
       uri: [Object],
       proxy: [Object],
       tunnel: true,
       agent: [Object],
       setHost: true,
       originalCookieHeader: undefined,
       _disableCookies: true,
       _jar: undefined,
       port: 443,
       host: 'celtia.zetapush.com',
       path: '/zbo/auth/whoami',
       httpModule: [Object],
       _started: true,
       href: 'https://celtia.zetapush.com/zbo/auth/whoami',
       req: [Object],
       ntick: true,
       response: [Circular],
       originalHost: 'celtia.zetapush.com',
       originalHostHeaderName: 'host',
       responseContent: [Circular],
       _destdata: true,
       _ended: true,
       _callbackCalled: true },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    read: [Function],
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' } ] ‌
[LOG] POST [ 'https://celtia.zetapush.com:/zbo//auth/login',
  IncomingMessage {
    _readableState: 
     ReadableState {
       objectMode: false,
       highWaterMark: 16384,
       buffer: [Object],
       length: 0,
       pipes: null,
       pipesCount: 0,
       flowing: true,
       ended: true,
       endEmitted: true,
       reading: false,
       sync: false,
       needReadable: false,
       emittedReadable: false,
       readableListening: false,
       resumeScheduled: false,
       destroyed: false,
       defaultEncoding: 'utf8',
       awaitDrain: 0,
       readingMore: false,
       decoder: null,
       encoding: null },
    readable: false,
    domain: null,
    _events: 
     { end: [Array],
       close: [Array],
       data: [Function],
       error: [Function] },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 297,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 210,
       [Symbol(bytesRead)]: 318 },
    connection: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 297,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 210,
       [Symbol(bytesRead)]: 318 },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: 
     { 'x-content-type-options': 'nosniff',
       'x-xss-protection': '1; mode=block',
       'cache-control': 'no-cache, no-store, max-age=0, must-revalidate',
       pragma: 'no-cache',
       expires: '0',
       'x-frame-options': 'DENY',
       connection: 'close',
       server: 'Jetty(9.2.8.v20150217)' },
    rawHeaders: 
     [ 'X-Content-Type-Options',
       'nosniff',
       'X-XSS-Protection',
       '1; mode=block',
       'Cache-Control',
       'no-cache, no-store, max-age=0, must-revalidate',
       'Pragma',
       'no-cache',
       'Expires',
       '0',
       'X-Frame-Options',
       'DENY',
       'Connection',
       'close',
       'Server',
       'Jetty(9.2.8.v20150217)' ],
    trailers: {},
    rawTrailers: [],
    upgrade: false,
    url: '',
    method: null,
    statusCode: 403,
    statusMessage: 'Forbidden',
    client: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 297,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 210,
       [Symbol(bytesRead)]: 318 },
    _consuming: true,
    _dumped: false,
    req: 
     ClientRequest {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       output: [],
       outputEncodings: [],
       outputCallbacks: [],
       outputSize: 0,
       writable: true,
       _last: true,
       upgrading: false,
       chunkedEncoding: false,
       shouldKeepAlive: false,
       useChunkedEncodingByDefault: true,
       sendDate: false,
       _removedConnection: false,
       _removedContLen: false,
       _removedTE: false,
       _contentLength: null,
       _hasBody: true,
       _trailer: '',
       finished: true,
       _headerSent: true,
       socket: [Object],
       connection: [Object],
       _header: 'POST /zbo//auth/login HTTP/1.1\r\nContent-Type: application/json;charset=UTF-8\r\nX-Authorization: {"username":"accountnotexists@zetapush.com","password":"password"}\r\nhost: celtia.zetapush.com\r\ncontent-length: 66\r\nConnection: close\r\n\r\n',
       _onPendingData: [Function: noopPendingOutput],
       agent: [Object],
       socketPath: undefined,
       timeout: undefined,
       method: 'POST',
       path: '/zbo//auth/login',
       _ended: true,
       res: [Circular],
       aborted: undefined,
       timeoutCb: null,
       upgradeOrConnect: false,
       parser: null,
       maxHeadersCount: null,
       [Symbol(outHeadersKey)]: [Object] },
    request: 
     Request {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       body: '{"username":"accountnotexists@zetapush.com","password":"password"}',
       headers: [Object],
       method: 'POST',
       callback: [Function],
       readable: true,
       writable: true,
       explicitMethod: true,
       _qs: [Object],
       _auth: [Object],
       _oauth: [Object],
       _multipart: [Object],
       _redirect: [Object],
       _tunnel: [Object],
       setHeader: [Function],
       hasHeader: [Function],
       getHeader: [Function],
       removeHeader: [Function],
       localAddress: undefined,
       pool: {},
       dests: [],
       __isRequestRequest: true,
       _callback: [Function],
       uri: [Object],
       proxy: [Object],
       tunnel: true,
       agent: [Object],
       setHost: true,
       originalCookieHeader: undefined,
       _disableCookies: true,
       _jar: undefined,
       port: 443,
       host: 'celtia.zetapush.com',
       path: '/zbo//auth/login',
       httpModule: [Object],
       _started: true,
       href: 'https://celtia.zetapush.com/zbo//auth/login',
       req: [Object],
       ntick: true,
       response: [Circular],
       originalHost: 'celtia.zetapush.com',
       originalHostHeaderName: 'host',
       responseContent: [Circular],
       _destdata: true,
       _ended: true,
       _callbackCalled: true },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    read: [Function],
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' } ] ‌
[TRACE] login failed [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}',
    request: 
     { body: '{"username":"accountnotexists@zetapush.com","password":"password"}',
       headers: [Object],
       method: 'POST',
       url: 'https://celtia.zetapush.com:/zbo//auth/login' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotexists@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
[TRACE] account doesn't exists  ‌
[TRACE] Analyze done => errorCode=ACCOUNT-03. Displaying help for this code  ‌
[TRACE] Display help message for ACCOUNT-03  ‌
[TRACE] Cache expired=false  ‌
[TRACE] Use cached troubleshoot for ACCOUNT-03  ‌
(?) 
Wrong login or password.

Check your credentials

TODO: https://github.com/zetapush/documentation/issues/20


Did you click the confirmation link in the email ?

When you created your account, we sent you an email with a confirmation link.
You must click the confirmation link in order to validate your account and be able to use ZetaPush services.
This confirmation link is valid for 7 days.

If you didn't click in time, your account has been deleted and you must create it again.

Are you using the right platform ?

TODO: https://github.com/zetapush/documentation/issues/21
  ‌
 
INFO    2018-07-13T14:18:07.432Z - COMMAND            | zetaPush(testing-projects/empty-app) -> [npm run deploy -- -vvv] 
ERROR   2018-07-13T14:18:09.100Z - COMMAND            | zetaPush(testing-projects/empty-app) -> [npm run deploy -- -vvv] {"exitCode":53}
INFO    2018-07-13T14:18:09.101Z - USER ACTION        | >>> User action DONE: zeta push from console 
ERROR   2018-07-13T14:18:09.099Z - PROCESS STDOUT/STD | 
[ERROR] Push failed [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}',
    request: 
     { body: undefined,
       headers: [Object],
       method: 'GET',
       url: 'https://celtia.zetapush.com:/zbo/orga/cluster/list' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotexists@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
npm ERR! code ELIFECYCLE
npm ERR! errno 53
npm ERR! empty-project@0.0.1 deploy: `zeta push "-vvv"`
npm ERR! Exit status 53
npm ERR! 
npm ERR! Failed at the empty-project@0.0.1 deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/aurelien/.npm/_logs/2018-07-13T14_18_09_091Z-debug.log
 
DEBUG   2018-07-13T14:18:09.119Z - GIVEN              | >> Apply Given 
INFO    2018-07-13T14:18:09.122Z - COMMAND            | npmInstallLatestVersion(testing-projects/empty-app) 
INFO    2018-07-13T14:18:09.120Z - GIVEN              | >>> Given testing application: testing-projects/empty-app 
SILLY   2018-07-13T14:18:09.121Z - GIVEN              | GivenTestingApp:execute -> npmInstallLatestVersion(testing-projects/empty-app) 
SILLY   2018-07-13T14:18:09.486Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] 
SILLY   2018-07-13T14:18:14.179Z - PROCESS STDOUT/STD | 

> get-cursor-position@1.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position
> node-gyp rebuild

make : on entre dans le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »
  CXX(target) Release/obj.target/pos/src/pos.o
  SOLINK_MODULE(target) Release/obj.target/pos.node
  COPY Release/pos.node
make : on quitte le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »

> bufferutil@3.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/bufferutil
> prebuild-install || node-gyp rebuild


> utf-8-validate@4.0.2 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/utf-8-validate
> prebuild-install || node-gyp rebuild

+ @zetapush/cli@0.30.0-alpha.743bc6df
added 208 packages in 4.247s 
SILLY   2018-07-13T14:18:14.178Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] ->  
SILLY   2018-07-13T14:18:14.179Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] 
SILLY   2018-07-13T14:18:16.129Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] ->  
SILLY   2018-07-13T14:18:16.131Z - COMMAND            | symlinkLocalDependencies(testing-projects/empty-app) 
WARN    2018-07-13T14:18:14.179Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^ 
SILLY   2018-07-13T14:18:16.129Z - PROCESS STDOUT/STD | 
+ @zetapush/platform@0.30.0-alpha.743bc6df
updated 1 package in 1.562s 
WARN    2018-07-13T14:18:16.130Z - PROCESS STDOUT/STD | 
 
SILLY   2018-07-13T14:18:16.429Z - GIVEN              | GivenTestingApp:execute -> setAccountToZetarc(testing-projects/empty-app, {"developerLogin":"accountnotexists@zetapush.com","developerPassword":"password"}) 
DEBUG   2018-07-13T14:18:16.431Z - COMMAND            | readZetarc(testing-projects/empty-app) 
DEBUG   2018-07-13T14:18:16.431Z - COMMAND            | readZetarc(testing-projects/empty-app) ->  
INFO    2018-07-13T14:18:16.432Z - USER ACTION        | >>> User action: zeta run from console 
ERROR   2018-07-13T14:18:18.089Z - PROCESS STDOUT/STD | 

> empty-project@0.0.1 start /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app
> zeta run "-vvv"

[LOG] Load and merge configuration with this priority order:
      1) command line arguments
      2) environment variables
      3) from file defined in ./.zetarc  ‌
[TRACE] Try to load conf from cli provess.argv  ‌
[TRACE] Try to load conf from process.env  ‌
[TRACE] Try to load conf from filesystem [ '.' ] ‌
[TRACE] Configuration loaded from file [ '.',
  { config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       developerLogin: 'accountnotexists@zetapush.com',
       developerPassword: 'password' },
    filepath: '/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/.zetarc' } ] ‌
[TRACE] Using default values  ‌
[TRACE] merged config [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'accountnotexists@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[TRACE] config is valid [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'accountnotexists@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/auth/whoami', undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/cluster/list',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/auth/whoami',
  IncomingMessage {
    _readableState: 
     ReadableState {
       objectMode: false,
       highWaterMark: 16384,
       buffer: [Object],
       length: 0,
       pipes: null,
       pipesCount: 0,
       flowing: true,
       ended: true,
       endEmitted: true,
       reading: false,
       sync: false,
       needReadable: false,
       emittedReadable: false,
       readableListening: false,
       resumeScheduled: false,
       destroyed: false,
       defaultEncoding: 'utf8',
       awaitDrain: 0,
       readingMore: false,
       decoder: null,
       encoding: null },
    readable: false,
    domain: null,
    _events: 
     { end: [Array],
       close: [Array],
       data: [Function],
       error: [Function] },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 210,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 168,
       [Symbol(bytesRead)]: 318 },
    connection: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 210,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 168,
       [Symbol(bytesRead)]: 318 },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: 
     { 'x-content-type-options': 'nosniff',
       'x-xss-protection': '1; mode=block',
       'cache-control': 'no-cache, no-store, max-age=0, must-revalidate',
       pragma: 'no-cache',
       expires: '0',
       'x-frame-options': 'DENY',
       connection: 'close',
       server: 'Jetty(9.2.8.v20150217)' },
    rawHeaders: 
     [ 'X-Content-Type-Options',
       'nosniff',
       'X-XSS-Protection',
       '1; mode=block',
       'Cache-Control',
       'no-cache, no-store, max-age=0, must-revalidate',
       'Pragma',
       'no-cache',
       'Expires',
       '0',
       'X-Frame-Options',
       'DENY',
       'Connection',
       'close',
       'Server',
       'Jetty(9.2.8.v20150217)' ],
    trailers: {},
    rawTrailers: [],
    upgrade: false,
    url: '',
    method: null,
    statusCode: 403,
    statusMessage: 'Forbidden',
    client: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 210,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 168,
       [Symbol(bytesRead)]: 318 },
    _consuming: true,
    _dumped: false,
    req: 
     ClientRequest {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       output: [],
       outputEncodings: [],
       outputCallbacks: [],
       outputSize: 0,
       writable: true,
       _last: true,
       upgrading: false,
       chunkedEncoding: false,
       shouldKeepAlive: false,
       useChunkedEncodingByDefault: false,
       sendDate: false,
       _removedConnection: false,
       _removedContLen: false,
       _removedTE: false,
       _contentLength: 0,
       _hasBody: true,
       _trailer: '',
       finished: true,
       _headerSent: true,
       socket: [Object],
       connection: [Object],
       _header: 'GET /zbo/auth/whoami HTTP/1.1\r\nContent-Type: application/json;charset=UTF-8\r\nX-Authorization: {"username":"accountnotexists@zetapush.com","password":"password"}\r\nhost: celtia.zetapush.com\r\nConnection: close\r\n\r\n',
       _onPendingData: [Function: noopPendingOutput],
       agent: [Object],
       socketPath: undefined,
       timeout: undefined,
       method: 'GET',
       path: '/zbo/auth/whoami',
       _ended: true,
       res: [Circular],
       aborted: undefined,
       timeoutCb: null,
       upgradeOrConnect: false,
       parser: null,
       maxHeadersCount: null,
       [Symbol(outHeadersKey)]: [Object] },
    request: 
     Request {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       headers: [Object],
       method: 'GET',
       callback: [Function],
       readable: true,
       writable: true,
       explicitMethod: true,
       _qs: [Object],
       _auth: [Object],
       _oauth: [Object],
       _multipart: [Object],
       _redirect: [Object],
       _tunnel: [Object],
       setHeader: [Function],
       hasHeader: [Function],
       getHeader: [Function],
       removeHeader: [Function],
       localAddress: undefined,
       pool: {},
       dests: [],
       __isRequestRequest: true,
       _callback: [Function],
       uri: [Object],
       proxy: [Object],
       tunnel: true,
       agent: [Object],
       setHost: true,
       originalCookieHeader: undefined,
       _disableCookies: true,
       _jar: undefined,
       port: 443,
       host: 'celtia.zetapush.com',
       path: '/zbo/auth/whoami',
       httpModule: [Object],
       _started: true,
       href: 'https://celtia.zetapush.com/zbo/auth/whoami',
       req: [Object],
       ntick: true,
       response: [Circular],
       originalHost: 'celtia.zetapush.com',
       originalHostHeaderName: 'host',
       responseContent: [Circular],
       _destdata: true,
       _ended: true,
       _callbackCalled: true },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    read: [Function],
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' } ] ‌
[TRACE] not config issue  ‌
[TRACE] not a network error  ‌
[TRACE] access denied [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}',
    request: 
     { body: undefined,
       headers: [Object],
       method: 'GET',
       url: 'https://celtia.zetapush.com:/zbo/auth/whoami' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotexists@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
[LOG] POST [ 'https://celtia.zetapush.com:/zbo//auth/login',
  '{"username":"accountnotexists@zetapush.com","password":"password"}' ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/cluster/list',
  IncomingMessage {
    _readableState: 
     ReadableState {
       objectMode: false,
       highWaterMark: 16384,
       buffer: [Object],
       length: 0,
       pipes: null,
       pipesCount: 0,
       flowing: true,
       ended: true,
       endEmitted: true,
       reading: false,
       sync: false,
       needReadable: false,
       emittedReadable: false,
       readableListening: false,
       resumeScheduled: false,
       destroyed: false,
       defaultEncoding: 'utf8',
       awaitDrain: 0,
       readingMore: false,
       decoder: null,
       encoding: null },
    readable: false,
    domain: null,
    _events: 
     { end: [Array],
       close: [Array],
       data: [Function],
       error: [Function] },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 216,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 159,
       [Symbol(bytesRead)]: 318 },
    connection: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 216,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 159,
       [Symbol(bytesRead)]: 318 },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: 
     { 'x-content-type-options': 'nosniff',
       'x-xss-protection': '1; mode=block',
       'cache-control': 'no-cache, no-store, max-age=0, must-revalidate',
       pragma: 'no-cache',
       expires: '0',
       'x-frame-options': 'DENY',
       connection: 'close',
       server: 'Jetty(9.2.8.v20150217)' },
    rawHeaders: 
     [ 'X-Content-Type-Options',
       'nosniff',
       'X-XSS-Protection',
       '1; mode=block',
       'Cache-Control',
       'no-cache, no-store, max-age=0, must-revalidate',
       'Pragma',
       'no-cache',
       'Expires',
       '0',
       'X-Frame-Options',
       'DENY',
       'Connection',
       'close',
       'Server',
       'Jetty(9.2.8.v20150217)' ],
    trailers: {},
    rawTrailers: [],
    upgrade: false,
    url: '',
    method: null,
    statusCode: 403,
    statusMessage: 'Forbidden',
    client: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 216,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 159,
       [Symbol(bytesRead)]: 318 },
    _consuming: true,
    _dumped: false,
    req: 
     ClientRequest {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       output: [],
       outputEncodings: [],
       outputCallbacks: [],
       outputSize: 0,
       writable: true,
       _last: true,
       upgrading: false,
       chunkedEncoding: false,
       shouldKeepAlive: false,
       useChunkedEncodingByDefault: false,
       sendDate: false,
       _removedConnection: false,
       _removedContLen: false,
       _removedTE: false,
       _contentLength: 0,
       _hasBody: true,
       _trailer: '',
       finished: true,
       _headerSent: true,
       socket: [Object],
       connection: [Object],
       _header: 'GET /zbo/orga/cluster/list HTTP/1.1\r\nContent-Type: application/json;charset=UTF-8\r\nX-Authorization: {"username":"accountnotexists@zetapush.com","password":"password"}\r\nhost: celtia.zetapush.com\r\nConnection: close\r\n\r\n',
       _onPendingData: [Function: noopPendingOutput],
       agent: [Object],
       socketPath: undefined,
       timeout: undefined,
       method: 'GET',
       path: '/zbo/orga/cluster/list',
       _ended: true,
       res: [Circular],
       aborted: undefined,
       timeoutCb: null,
       upgradeOrConnect: false,
       parser: null,
       maxHeadersCount: null,
       [Symbol(outHeadersKey)]: [Object] },
    request: 
     Request {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       headers: [Object],
       method: 'GET',
       callback: [Function],
       readable: true,
       writable: true,
       explicitMethod: true,
       _qs: [Object],
       _auth: [Object],
       _oauth: [Object],
       _multipart: [Object],
       _redirect: [Object],
       _tunnel: [Object],
       setHeader: [Function],
       hasHeader: [Function],
       getHeader: [Function],
       removeHeader: [Function],
       localAddress: undefined,
       pool: {},
       dests: [],
       __isRequestRequest: true,
       _callback: [Function],
       uri: [Object],
       proxy: [Object],
       tunnel: true,
       agent: [Object],
       setHost: true,
       originalCookieHeader: undefined,
       _disableCookies: true,
       _jar: undefined,
       port: 443,
       host: 'celtia.zetapush.com',
       path: '/zbo/orga/cluster/list',
       httpModule: [Object],
       _started: true,
       href: 'https://celtia.zetapush.com/zbo/orga/cluster/list',
       req: [Object],
       ntick: true,
       response: [Circular],
       originalHost: 'celtia.zetapush.com',
       originalHostHeaderName: 'host',
       responseContent: [Circular],
       _destdata: true,
       _ended: true,
       _callbackCalled: true },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    read: [Function],
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' } ] ‌
[LOG] POST [ 'https://celtia.zetapush.com:/zbo//auth/login',
  IncomingMessage {
    _readableState: 
     ReadableState {
       objectMode: false,
       highWaterMark: 16384,
       buffer: [Object],
       length: 0,
       pipes: null,
       pipesCount: 0,
       flowing: true,
       ended: true,
       endEmitted: true,
       reading: false,
       sync: false,
       needReadable: false,
       emittedReadable: false,
       readableListening: false,
       resumeScheduled: false,
       destroyed: false,
       defaultEncoding: 'utf8',
       awaitDrain: 0,
       readingMore: false,
       decoder: null,
       encoding: null },
    readable: false,
    domain: null,
    _events: 
     { end: [Array],
       close: [Array],
       data: [Function],
       error: [Function] },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 297,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 210,
       [Symbol(bytesRead)]: 318 },
    connection: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 297,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 210,
       [Symbol(bytesRead)]: 318 },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    headers: 
     { 'x-content-type-options': 'nosniff',
       'x-xss-protection': '1; mode=block',
       'cache-control': 'no-cache, no-store, max-age=0, must-revalidate',
       pragma: 'no-cache',
       expires: '0',
       'x-frame-options': 'DENY',
       connection: 'close',
       server: 'Jetty(9.2.8.v20150217)' },
    rawHeaders: 
     [ 'X-Content-Type-Options',
       'nosniff',
       'X-XSS-Protection',
       '1; mode=block',
       'Cache-Control',
       'no-cache, no-store, max-age=0, must-revalidate',
       'Pragma',
       'no-cache',
       'Expires',
       '0',
       'X-Frame-Options',
       'DENY',
       'Connection',
       'close',
       'Server',
       'Jetty(9.2.8.v20150217)' ],
    trailers: {},
    rawTrailers: [],
    upgrade: false,
    url: '',
    method: null,
    statusCode: 403,
    statusMessage: 'Forbidden',
    client: 
     TLSSocket {
       _tlsOptions: [Object],
       _secureEstablished: true,
       _securePending: false,
       _newSessionPending: false,
       _controlReleased: true,
       _SNICallback: null,
       servername: 'celtia.zetapush.com',
       npnProtocol: undefined,
       alpnProtocol: false,
       authorized: true,
       authorizationError: null,
       encrypted: true,
       _events: [Object],
       _eventsCount: 10,
       connecting: false,
       _hadError: false,
       _handle: null,
       _parent: [Object],
       _host: null,
       _readableState: [Object],
       readable: false,
       domain: null,
       _maxListeners: undefined,
       _writableState: [Object],
       writable: false,
       allowHalfOpen: false,
       _bytesDispatched: 297,
       _sockname: null,
       _pendingData: null,
       _pendingEncoding: '',
       server: undefined,
       _server: null,
       ssl: null,
       _requestCert: true,
       _rejectUnauthorized: true,
       parser: null,
       _httpMessage: [Object],
       write: [Function: writeAfterFIN],
       _idleNext: null,
       _idlePrev: null,
       _idleTimeout: -1,
       [Symbol(asyncId)]: 210,
       [Symbol(bytesRead)]: 318 },
    _consuming: true,
    _dumped: false,
    req: 
     ClientRequest {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       output: [],
       outputEncodings: [],
       outputCallbacks: [],
       outputSize: 0,
       writable: true,
       _last: true,
       upgrading: false,
       chunkedEncoding: false,
       shouldKeepAlive: false,
       useChunkedEncodingByDefault: true,
       sendDate: false,
       _removedConnection: false,
       _removedContLen: false,
       _removedTE: false,
       _contentLength: null,
       _hasBody: true,
       _trailer: '',
       finished: true,
       _headerSent: true,
       socket: [Object],
       connection: [Object],
       _header: 'POST /zbo//auth/login HTTP/1.1\r\nContent-Type: application/json;charset=UTF-8\r\nX-Authorization: {"username":"accountnotexists@zetapush.com","password":"password"}\r\nhost: celtia.zetapush.com\r\ncontent-length: 66\r\nConnection: close\r\n\r\n',
       _onPendingData: [Function: noopPendingOutput],
       agent: [Object],
       socketPath: undefined,
       timeout: undefined,
       method: 'POST',
       path: '/zbo//auth/login',
       _ended: true,
       res: [Circular],
       aborted: undefined,
       timeoutCb: null,
       upgradeOrConnect: false,
       parser: null,
       maxHeadersCount: null,
       [Symbol(outHeadersKey)]: [Object] },
    request: 
     Request {
       domain: null,
       _events: [Object],
       _eventsCount: 5,
       _maxListeners: undefined,
       body: '{"username":"accountnotexists@zetapush.com","password":"password"}',
       headers: [Object],
       method: 'POST',
       callback: [Function],
       readable: true,
       writable: true,
       explicitMethod: true,
       _qs: [Object],
       _auth: [Object],
       _oauth: [Object],
       _multipart: [Object],
       _redirect: [Object],
       _tunnel: [Object],
       setHeader: [Function],
       hasHeader: [Function],
       getHeader: [Function],
       removeHeader: [Function],
       localAddress: undefined,
       pool: {},
       dests: [],
       __isRequestRequest: true,
       _callback: [Function],
       uri: [Object],
       proxy: [Object],
       tunnel: true,
       agent: [Object],
       setHost: true,
       originalCookieHeader: undefined,
       _disableCookies: true,
       _jar: undefined,
       port: 443,
       host: 'celtia.zetapush.com',
       path: '/zbo//auth/login',
       httpModule: [Object],
       _started: true,
       href: 'https://celtia.zetapush.com/zbo//auth/login',
       req: [Object],
       ntick: true,
       response: [Circular],
       originalHost: 'celtia.zetapush.com',
       originalHostHeaderName: 'host',
       responseContent: [Circular],
       _destdata: true,
       _ended: true,
       _callbackCalled: true },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    read: [Function],
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' } ] ‌
[TRACE] login failed [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}',
    request: 
     { body: '{"username":"accountnotexists@zetapush.com","password":"password"}',
       headers: [Object],
       method: 'POST',
       url: 'https://celtia.zetapush.com:/zbo//auth/login' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotexists@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
[TRACE] account doesn't exists  ‌
[TRACE] Analyze done => errorCode=ACCOUNT-03. Displaying help for this code  ‌
[TRACE] Display help message for ACCOUNT-03  ‌
[TRACE] Cache expired=false  ‌
[TRACE] Use cached troubleshoot for ACCOUNT-03  ‌
(?) 
Wrong login or password.

Check your credentials

TODO: https://github.com/zetapush/documentation/issues/20


Did you click the confirmation link in the email ?

When you created your account, we sent you an email with a confirmation link.
You must click the confirmation link in order to validate your account and be able to use ZetaPush services.
This confirmation link is valid for 7 days.

If you didn't click in time, your account has been deleted and you must create it again.

Are you using the right platform ?

TODO: https://github.com/zetapush/documentation/issues/21
  ‌
 
INFO    2018-07-13T14:18:16.433Z - COMMAND            | zetaRun(testing-projects/empty-app) -> [npm run start -- -vvv] 
ERROR   2018-07-13T14:18:18.091Z - COMMAND            | zetaRun(testing-projects/empty-app) -> [npm run start -- -vvv] {"exitCode":53}
INFO    2018-07-13T14:18:18.092Z - USER ACTION        | >>> User action DONE: zeta run from console 
ERROR   2018-07-13T14:18:18.091Z - PROCESS STDOUT/STD | 
[ERROR] Run failed [ { response: 
     IncomingMessage {
       _readableState: [Object],
       readable: false,
       domain: null,
       _events: [Object],
       _eventsCount: 4,
       _maxListeners: undefined,
       socket: [Object],
       connection: [Object],
       httpVersionMajor: 1,
       httpVersionMinor: 1,
       httpVersion: '1.1',
       complete: true,
       headers: [Object],
       rawHeaders: [Array],
       trailers: {},
       rawTrailers: [],
       upgrade: false,
       url: '',
       method: null,
       statusCode: 403,
       statusMessage: 'Forbidden',
       client: [Object],
       _consuming: true,
       _dumped: false,
       req: [Object],
       request: [Object],
       toJSON: [Function: responseToJSON],
       caseless: [Object],
       read: [Function],
       body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}' },
    statusCode: 403,
    body: '{"code":"ACCESS_FORBIDDEN","message":"Access is forbidden"}',
    request: 
     { body: undefined,
       headers: [Object],
       method: 'GET',
       url: 'https://celtia.zetapush.com:/zbo/auth/whoami' },
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'accountnotexists@zetapush.com',
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
npm ERR! code ELIFECYCLE
npm ERR! errno 53
npm ERR! empty-project@0.0.1 start: `zeta run "-vvv"`
npm ERR! Exit status 53
npm ERR! 
npm ERR! Failed at the empty-project@0.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/aurelien/.npm/_logs/2018-07-13T14_18_18_083Z-debug.log
 
DEBUG   2018-07-13T14:18:18.133Z - GIVEN              | >> Apply Given 
INFO    2018-07-13T14:18:18.135Z - GIVEN              | >>> Given new application: .generated-projects/no-appname 
SILLY   2018-07-13T14:18:18.135Z - GIVEN              | GivenNewApp:execute -> rm(.generated-projects/no-appname) 
SILLY   2018-07-13T14:18:18.977Z - GIVEN              | GivenNewApp:execute -> npmInit(aurelien.baudet+celtia@zetapush.com,
          password, .generated-projects/no-appname) 
INFO    2018-07-13T14:18:18.978Z - COMMAND            | npmInit(aurelien.baudet+celtia@zetapush.com, password, .generated-projects/no-appname) 
DEBUG   2018-07-13T14:18:18.979Z - COMMAND            | npmVersion() -> [npm --version] 
DEBUG   2018-07-13T14:18:43.824Z - COMMAND            | npmInit() -> [npx @zetapush/create@canary no-appname --force-current-version --developer-login xxx --developer-password xxx] 
WARN    2018-07-13T14:19:01.229Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^
 
SILLY   2018-07-13T14:19:02.100Z - PROCESS STDOUT/STD | 
__dirname= /home/aurelien/.npm/_npx/29740/lib/node_modules/@zetapush/create
 
SILLY   2018-07-13T14:19:02.328Z - PROCESS STDOUT/STD | 

 
SILLY   2018-07-13T14:19:02.330Z - PROCESS STDOUT/STD | 
Creating a new ZetaPush app in /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/.generated-projects/no-appname.
 
SILLY   2018-07-13T14:19:02.332Z - PROCESS STDOUT/STD | 

 
SILLY   2018-07-13T14:19:02.335Z - PROCESS STDOUT/STD | 
Installing packages. This might take a couple of minutes.
 
SILLY   2018-07-13T14:19:02.337Z - PROCESS STDOUT/STD | 
Installing @zetapush/cli@0.30.0-alpha.743bc6df, @zetapush/platform@0.30.0-alpha.743bc6df, typescript@latest.
 
SILLY   2018-07-13T14:19:02.339Z - PROCESS STDOUT/STD | 

 
SILLY   2018-07-13T14:19:07.669Z - PROCESS STDOUT/STD | 

> get-cursor-position@1.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/.generated-projects/no-appname/node_modules/get-cursor-position
> node-gyp rebuild

 
SILLY   2018-07-13T14:19:09.647Z - PROCESS STDOUT/STD | 
make : on entre dans le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/.generated-projects/no-appname/node_modules/get-cursor-position/build »
 
SILLY   2018-07-13T14:19:09.649Z - PROCESS STDOUT/STD | 
  CXX(target) Release/obj.target/pos/src/pos.o
 
WARN    2018-07-13T14:19:10.466Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^
 
SILLY   2018-07-13T14:19:10.557Z - PROCESS STDOUT/STD | 
  SOLINK_MODULE(target) Release/obj.target/pos.node
 
SILLY   2018-07-13T14:19:10.725Z - PROCESS STDOUT/STD | 
  COPY Release/pos.node
 
SILLY   2018-07-13T14:19:10.732Z - PROCESS STDOUT/STD | 
make : on quitte le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/.generated-projects/no-appname/node_modules/get-cursor-position/build »
 
SILLY   2018-07-13T14:19:10.757Z - PROCESS STDOUT/STD | 

> bufferutil@3.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/.generated-projects/no-appname/node_modules/bufferutil
> prebuild-install || node-gyp rebuild

 
SILLY   2018-07-13T14:19:10.917Z - PROCESS STDOUT/STD | 

> utf-8-validate@4.0.2 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/.generated-projects/no-appname/node_modules/utf-8-validate
> prebuild-install || node-gyp rebuild

 
SILLY   2018-07-13T14:19:11.170Z - PROCESS STDOUT/STD | 
+ @zetapush/cli@0.30.0-alpha.743bc6df
+ @zetapush/platform@0.30.0-alpha.743bc6df
+ typescript@2.9.2
added 208 packages in 8.334s
 
SILLY   2018-07-13T14:19:11.187Z - PROCESS STDOUT/STD | 

 
SILLY   2018-07-13T14:19:11.188Z - PROCESS STDOUT/STD | 
Success! Created no-appname at /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/.generated-projects/no-appname
Inside that directory, you can run several commands:

  npm run start
    Starts the development server.

  npm run deploy
    Deploy your application on ZetaPush platform.

We suggest that you begin by typing:

  cd no-appname
  npm run start

Enjoy, Celtia is on your back !
 
SILLY   2018-07-13T14:19:11.507Z - COMMAND            | symlinkLocalDependencies(.generated-projects/no-appname) 
SILLY   2018-07-13T14:19:11.508Z - GIVEN              | GivenNewApp:execute -> setAppNameToZetarc(.generated-projects/no-appname, ) 
DEBUG   2018-07-13T14:19:11.548Z - COMMAND            | readZetarc(.generated-projects/no-appname) 
DEBUG   2018-07-13T14:19:11.677Z - COMMAND            | readZetarc(.generated-projects/no-appname) ->  
INFO    2018-07-13T14:19:11.678Z - USER ACTION        | >>> User action: 1) zeta push from console 
SILLY   2018-07-13T14:20:06.350Z - PROCESS STDOUT/STD | 

> no-appname@0.1.0 deploy /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/.generated-projects/no-appname
> zeta push "-vvv"

[LOG] Load and merge configuration with this priority order:
      1) command line arguments
      2) environment variables
      3) from file defined in ./.zetarc  ‌
[TRACE] Try to load conf from cli provess.argv  ‌
[TRACE] Try to load conf from process.env  ‌
[TRACE] Try to load conf from filesystem [ '.' ] ‌
[TRACE] Configuration loaded from file [ '.',
  { config: 
     { developerLogin: 'aurelien.baudet+celtia@zetapush.com',
       developerPassword: 'password',
       platformUrl: 'https://celtia.zetapush.com/zbo/pub/business' },
    filepath: '/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/.generated-projects/no-appname/.zetarc' } ] ‌
[TRACE] Using default values  ‌
[TRACE] merged config [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'aurelien.baudet+celtia@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[TRACE] config is valid [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'aurelien.baudet+celtia@zetapush.com',
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/auth/whoami', undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/cluster/list',
  undefined ] ‌
[LOG] POST [ 'https://celtia.zetapush.com:/zbo/orga/business/create',
  '{"orgaId":418,"cluster":1,"name":"app@y720d5t","description":"app"}' ] ‌
[LOG] Save config file [ '.zetarc' ] ‌
[LOG] Execute command <push> .  ‌
[LOG] Uploading [ '/tmp/1531491554979.zip' ] ‌
[LOG] Upload archive [ 'https://celtia.zetapush.com:/zbo/orga/recipe/cook' ] ‌
[TRACE] credentials [ 'aurelien.baudet+celtia@zetapush.com',
  'password',
  'https://celtia.zetapush.com/zbo/pub/business',
  'Qv9yj7sh' ] ‌
[LOG] Uploaded [ 'Pd0eHqGiCL7JwZMS' ] ‌
[LOG] Progression  ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/recipe/status/Qv9yj7sh/Pd0eHqGiCL7JwZMS',
  undefined ] ‌
[LOG] GET [ 'https://celtia.zetapush.com:/zbo/orga/business/live/Qv9yj7sh',
  undefined ] ‌
[LOG] Application status  ‌
[INFO] Your frontend application front is available at http://front-qv9yj7sh.celtia.zetapush.app:80/  ‌ 
INFO    2018-07-13T14:19:11.678Z - COMMAND            | zetaPush(.generated-projects/no-appname) -> [npm run deploy -- -vvv] 
SILLY   2018-07-13T14:20:06.348Z - COMMAND            | zetaRun(.generated-projects/no-appname) -> [npm run deploy -- -vvv] ->  
DEBUG   2018-07-13T14:20:06.355Z - COMMAND            | readZetarc(.generated-projects/no-appname) 
INFO    2018-07-13T14:20:06.353Z - USER ACTION        | >>> User action DONE: 1) zeta push from console 
WARN    2018-07-13T14:20:06.352Z - PROCESS STDOUT/STD | 
 
DEBUG   2018-07-13T14:20:06.410Z - COMMAND            | readZetarc(.generated-projects/no-appname) ->  
INFO    2018-07-13T14:20:06.411Z - USER ACTION        | >>> User action: 2) call hello from front 
DEBUG   2018-07-13T14:20:06.435Z - FRONT USER ACTION  | Connecting to worker... 
DEBUG   2018-07-13T14:20:07.000Z - FRONT USER ACTION  | Connected to worker 
DEBUG   2018-07-13T14:20:07.001Z - FRONT USER ACTION  | Api instance created 
INFO    2018-07-13T14:20:07.123Z - USER ACTION        | >>> User action DONE: 2) call hello from front 
INFO    2018-07-13T14:20:07.125Z - COMMAND            | nukeApp(.generated-projects/no-appname) 
SILLY   2018-07-13T14:20:07.125Z - COMMAND            | nukeApp(.generated-projects/no-appname) -> readZetarc(.generated-projects/no-appname) 
DEBUG   2018-07-13T14:20:07.126Z - COMMAND            | readZetarc(.generated-projects/no-appname) 
DEBUG   2018-07-13T14:20:07.130Z - COMMAND            | readZetarc(.generated-projects/no-appname) ->  
SILLY   2018-07-13T14:20:07.131Z - COMMAND            | nukeApp(.generated-projects/no-appname) -> DELETE orga/business/nuke/Qv9yj7sh 
SILLY   2018-07-13T14:20:07.889Z - COMMAND            | nukeApp(.generated-projects/no-appname) -> DELETE orga/business/nuke/Qv9yj7sh ->  {"queued":"Request noted. Full eradication will take a few minutes"}
DEBUG   2018-07-13T14:20:07.907Z - GIVEN              | >> Apply Given 
INFO    2018-07-13T14:20:07.909Z - COMMAND            | npmInstallLatestVersion(testing-projects/empty-app) 
INFO    2018-07-13T14:20:07.908Z - GIVEN              | >>> Given testing application: testing-projects/empty-app 
SILLY   2018-07-13T14:20:07.909Z - GIVEN              | GivenTestingApp:execute -> npmInstallLatestVersion(testing-projects/empty-app) 
SILLY   2018-07-13T14:20:08.354Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] 
SILLY   2018-07-13T14:20:13.307Z - PROCESS STDOUT/STD | 

> get-cursor-position@1.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position
> node-gyp rebuild

make : on entre dans le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »
  CXX(target) Release/obj.target/pos/src/pos.o
  SOLINK_MODULE(target) Release/obj.target/pos.node
  COPY Release/pos.node
make : on quitte le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »

> bufferutil@3.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/bufferutil
> prebuild-install || node-gyp rebuild


> utf-8-validate@4.0.2 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/utf-8-validate
> prebuild-install || node-gyp rebuild

+ @zetapush/cli@0.30.0-alpha.743bc6df
added 208 packages in 4.517s 
SILLY   2018-07-13T14:20:13.307Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] ->  
SILLY   2018-07-13T14:20:13.308Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] 
SILLY   2018-07-13T14:20:15.066Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] ->  
SILLY   2018-07-13T14:20:15.068Z - COMMAND            | symlinkLocalDependencies(testing-projects/empty-app) 
WARN    2018-07-13T14:20:13.308Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^ 
SILLY   2018-07-13T14:20:15.067Z - PROCESS STDOUT/STD | 
+ @zetapush/platform@0.30.0-alpha.743bc6df
updated 1 package in 1.37s 
WARN    2018-07-13T14:20:15.068Z - PROCESS STDOUT/STD | 
 
SILLY   2018-07-13T14:20:15.384Z - GIVEN              | GivenTestingApp:execute -> setAccountToZetarc(testing-projects/empty-app, {"developerLogin":"","developerPassword":"password"}) 
DEBUG   2018-07-13T14:20:15.386Z - COMMAND            | readZetarc(testing-projects/empty-app) 
DEBUG   2018-07-13T14:20:15.387Z - COMMAND            | readZetarc(testing-projects/empty-app) ->  
INFO    2018-07-13T14:20:15.387Z - USER ACTION        | >>> User action: zeta push from console 
ERROR   2018-07-13T14:20:16.314Z - PROCESS STDOUT/STD | 

> empty-project@0.0.1 deploy /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app
> zeta push "-vvv"

[LOG] Load and merge configuration with this priority order:
      1) command line arguments
      2) environment variables
      3) from file defined in ./.zetarc  ‌
[TRACE] Try to load conf from cli provess.argv  ‌
[TRACE] Try to load conf from process.env  ‌
[TRACE] Try to load conf from filesystem [ '.' ] ‌
[TRACE] Configuration loaded from file [ '.',
  { config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       developerPassword: 'password' },
    filepath: '/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/.zetarc' } ] ‌
[TRACE] Using default values  ‌
[TRACE] merged config [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: undefined,
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[TRACE] config is not valid [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: undefined,
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[TRACE] not config issue  ‌
[TRACE] not a network error  ‌
[TRACE] access denied [ { code: 'CONFIG_MISSING_REQUIRED_INFO',
    message: 'Missing required information',
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: undefined,
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
[TRACE] no developer login  ‌
[TRACE] Analyze done => errorCode=ACCOUNT-01. Displaying help for this code  ‌
[TRACE] Display help message for ACCOUNT-01  ‌
[TRACE] Cache expired=false  ‌
[TRACE] Use cached troubleshoot for ACCOUNT-01  ‌
(?) 
You didn't provide your ZetaPush account login.


I don't have ZetaPush account, how to create it ?

When you want to use ZetaPush Cloud services or push your custom Cloud services on ZetaPush, you need an account.

I use the CLI

TODO: https://github.com/zetapush/documentation/issues/14

I use ZetaPush console

TODO: https://github.com/zetapush/documentation/issues/15


I have a ZetaPush account, how to use it ?

TODO: https://github.com/zetapush/documentation/issues/16

I provide my crendentials manually

TODO: https://github.com/zetapush/documentation/issues/17

I use the CLI

TODO: https://github.com/zetapush/documentation/issues/18
  ‌
 
INFO    2018-07-13T14:20:15.388Z - COMMAND            | zetaPush(testing-projects/empty-app) -> [npm run deploy -- -vvv] 
ERROR   2018-07-13T14:20:16.318Z - COMMAND            | zetaPush(testing-projects/empty-app) -> [npm run deploy -- -vvv] {"exitCode":51}
INFO    2018-07-13T14:20:16.320Z - USER ACTION        | >>> User action DONE: zeta push from console 
ERROR   2018-07-13T14:20:16.316Z - PROCESS STDOUT/STD | 
[ERROR] Missing required information  ‌
[ERROR] Push failed [ { code: 'CONFIG_MISSING_REQUIRED_INFO',
    message: 'Missing required information',
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: undefined,
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
npm ERR! code ELIFECYCLE
npm ERR! errno 51
npm ERR! empty-project@0.0.1 deploy: `zeta push "-vvv"`
npm ERR! Exit status 51
npm ERR! 
npm ERR! Failed at the empty-project@0.0.1 deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/aurelien/.npm/_logs/2018-07-13T14_20_16_309Z-debug.log
 
DEBUG   2018-07-13T14:20:16.334Z - GIVEN              | >> Apply Given 
INFO    2018-07-13T14:20:16.336Z - COMMAND            | npmInstallLatestVersion(testing-projects/empty-app) 
INFO    2018-07-13T14:20:16.335Z - GIVEN              | >>> Given testing application: testing-projects/empty-app 
SILLY   2018-07-13T14:20:16.336Z - GIVEN              | GivenTestingApp:execute -> npmInstallLatestVersion(testing-projects/empty-app) 
SILLY   2018-07-13T14:20:16.758Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] 
SILLY   2018-07-13T14:20:21.537Z - PROCESS STDOUT/STD | 

> get-cursor-position@1.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position
> node-gyp rebuild

make : on entre dans le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »
  CXX(target) Release/obj.target/pos/src/pos.o
  SOLINK_MODULE(target) Release/obj.target/pos.node
  COPY Release/pos.node
make : on quitte le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »

> bufferutil@3.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/bufferutil
> prebuild-install || node-gyp rebuild


> utf-8-validate@4.0.2 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/utf-8-validate
> prebuild-install || node-gyp rebuild

+ @zetapush/cli@0.30.0-alpha.743bc6df
added 208 packages in 4.41s 
SILLY   2018-07-13T14:20:21.537Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] ->  
SILLY   2018-07-13T14:20:21.538Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] 
SILLY   2018-07-13T14:20:23.733Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] ->  
SILLY   2018-07-13T14:20:23.735Z - COMMAND            | symlinkLocalDependencies(testing-projects/empty-app) 
WARN    2018-07-13T14:20:21.538Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^ 
SILLY   2018-07-13T14:20:23.734Z - PROCESS STDOUT/STD | 
+ @zetapush/platform@0.30.0-alpha.743bc6df
updated 1 package in 1.802s 
WARN    2018-07-13T14:20:23.734Z - PROCESS STDOUT/STD | 
 
SILLY   2018-07-13T14:20:24.138Z - GIVEN              | GivenTestingApp:execute -> setAccountToZetarc(testing-projects/empty-app, {"developerLogin":"","developerPassword":"password"}) 
DEBUG   2018-07-13T14:20:24.140Z - COMMAND            | readZetarc(testing-projects/empty-app) 
DEBUG   2018-07-13T14:20:24.142Z - COMMAND            | readZetarc(testing-projects/empty-app) ->  
INFO    2018-07-13T14:20:24.145Z - USER ACTION        | >>> User action: zeta run from console 
ERROR   2018-07-13T14:20:25.116Z - PROCESS STDOUT/STD | 

> empty-project@0.0.1 start /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app
> zeta run "-vvv"

[LOG] Load and merge configuration with this priority order:
      1) command line arguments
      2) environment variables
      3) from file defined in ./.zetarc  ‌
[TRACE] Try to load conf from cli provess.argv  ‌
[TRACE] Try to load conf from process.env  ‌
[TRACE] Try to load conf from filesystem [ '.' ] ‌
[TRACE] Configuration loaded from file [ '.',
  { config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       developerPassword: 'password' },
    filepath: '/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/.zetarc' } ] ‌
[TRACE] Using default values  ‌
[TRACE] merged config [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: undefined,
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[TRACE] config is not valid [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: undefined,
    developerPassword: 'password',
    workerServiceId: undefined } ] ‌
[TRACE] not config issue  ‌
[TRACE] not a network error  ‌
[TRACE] access denied [ { code: 'CONFIG_MISSING_REQUIRED_INFO',
    message: 'Missing required information',
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: undefined,
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
[TRACE] no developer login  ‌
[TRACE] Analyze done => errorCode=ACCOUNT-01. Displaying help for this code  ‌
[TRACE] Display help message for ACCOUNT-01  ‌
[TRACE] Cache expired=false  ‌
[TRACE] Use cached troubleshoot for ACCOUNT-01  ‌
(?) 
You didn't provide your ZetaPush account login.


I don't have ZetaPush account, how to create it ?

When you want to use ZetaPush Cloud services or push your custom Cloud services on ZetaPush, you need an account.

I use the CLI

TODO: https://github.com/zetapush/documentation/issues/14

I use ZetaPush console

TODO: https://github.com/zetapush/documentation/issues/15


I have a ZetaPush account, how to use it ?

TODO: https://github.com/zetapush/documentation/issues/16

I provide my crendentials manually

TODO: https://github.com/zetapush/documentation/issues/17

I use the CLI

TODO: https://github.com/zetapush/documentation/issues/18
  ‌
 
INFO    2018-07-13T14:20:24.146Z - COMMAND            | zetaRun(testing-projects/empty-app) -> [npm run start -- -vvv] 
ERROR   2018-07-13T14:20:25.117Z - COMMAND            | zetaRun(testing-projects/empty-app) -> [npm run start -- -vvv] {"exitCode":51}
INFO    2018-07-13T14:20:25.118Z - USER ACTION        | >>> User action DONE: zeta run from console 
ERROR   2018-07-13T14:20:25.117Z - PROCESS STDOUT/STD | 
[ERROR] Missing required information  ‌
[ERROR] Run failed [ { code: 'CONFIG_MISSING_REQUIRED_INFO',
    message: 'Missing required information',
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: undefined,
       developerPassword: 'password',
       workerServiceId: undefined } } ] ‌
npm ERR! code ELIFECYCLE
npm ERR! errno 51
npm ERR! empty-project@0.0.1 start: `zeta run "-vvv"`
npm ERR! Exit status 51
npm ERR! 
npm ERR! Failed at the empty-project@0.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/aurelien/.npm/_logs/2018-07-13T14_20_25_109Z-debug.log
 
DEBUG   2018-07-13T14:20:25.150Z - GIVEN              | >> Apply Given 
INFO    2018-07-13T14:20:25.152Z - COMMAND            | npmInstallLatestVersion(testing-projects/empty-app) 
INFO    2018-07-13T14:20:25.151Z - GIVEN              | >>> Given testing application: testing-projects/empty-app 
SILLY   2018-07-13T14:20:25.152Z - GIVEN              | GivenTestingApp:execute -> npmInstallLatestVersion(testing-projects/empty-app) 
SILLY   2018-07-13T14:20:25.522Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] 
SILLY   2018-07-13T14:20:30.472Z - PROCESS STDOUT/STD | 

> get-cursor-position@1.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position
> node-gyp rebuild

make : on entre dans le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »
  CXX(target) Release/obj.target/pos/src/pos.o
  SOLINK_MODULE(target) Release/obj.target/pos.node
  COPY Release/pos.node
make : on quitte le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »

> bufferutil@3.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/bufferutil
> prebuild-install || node-gyp rebuild


> utf-8-validate@4.0.2 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/utf-8-validate
> prebuild-install || node-gyp rebuild

+ @zetapush/cli@0.30.0-alpha.743bc6df
added 208 packages in 4.59s 
SILLY   2018-07-13T14:20:30.471Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] ->  
SILLY   2018-07-13T14:20:30.473Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] 
SILLY   2018-07-13T14:20:32.286Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] ->  
SILLY   2018-07-13T14:20:32.288Z - COMMAND            | symlinkLocalDependencies(testing-projects/empty-app) 
WARN    2018-07-13T14:20:30.472Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^ 
SILLY   2018-07-13T14:20:32.287Z - PROCESS STDOUT/STD | 
+ @zetapush/platform@0.30.0-alpha.743bc6df
updated 1 package in 1.458s 
WARN    2018-07-13T14:20:32.287Z - PROCESS STDOUT/STD | 
 
SILLY   2018-07-13T14:20:32.606Z - GIVEN              | GivenTestingApp:execute -> setAccountToZetarc(testing-projects/empty-app, {"developerLogin":"user@zetapush.com","developerPassword":""}) 
DEBUG   2018-07-13T14:20:32.609Z - COMMAND            | readZetarc(testing-projects/empty-app) 
DEBUG   2018-07-13T14:20:32.610Z - COMMAND            | readZetarc(testing-projects/empty-app) ->  
INFO    2018-07-13T14:20:32.611Z - USER ACTION        | >>> User action: zeta push from console 
ERROR   2018-07-13T14:20:33.545Z - PROCESS STDOUT/STD | 

> empty-project@0.0.1 deploy /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app
> zeta push "-vvv"

[LOG] Load and merge configuration with this priority order:
      1) command line arguments
      2) environment variables
      3) from file defined in ./.zetarc  ‌
[TRACE] Try to load conf from cli provess.argv  ‌
[TRACE] Try to load conf from process.env  ‌
[TRACE] Try to load conf from filesystem [ '.' ] ‌
[TRACE] Configuration loaded from file [ '.',
  { config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       developerLogin: 'user@zetapush.com' },
    filepath: '/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/.zetarc' } ] ‌
[TRACE] Using default values  ‌
[TRACE] merged config [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'user@zetapush.com',
    developerPassword: undefined,
    workerServiceId: undefined } ] ‌
[TRACE] config is not valid [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'user@zetapush.com',
    developerPassword: '',
    workerServiceId: undefined } ] ‌
[TRACE] not config issue  ‌
[TRACE] not a network error  ‌
[TRACE] access denied [ { code: 'CONFIG_MISSING_REQUIRED_INFO',
    message: 'Missing required information',
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'user@zetapush.com',
       developerPassword: '',
       workerServiceId: undefined } } ] ‌
[TRACE] no developer password  ‌
[TRACE] Analyze done => errorCode=ACCOUNT-02. Displaying help for this code  ‌
[TRACE] Display help message for ACCOUNT-02  ‌
[TRACE] Cache expired=false  ‌
[TRACE] Use cached troubleshoot for ACCOUNT-02  ‌
(?) 
You did provide your ZetaPush account login but not your password.


I provide my password in .zetarc file

Edit the .zetarc file to add developerPassword with your password:

{                                         
  "developerLogin": "<your login>",       
  "developerPassword": "<your password>"  
}                                        

   | Sensitive information leak
/!\| You must check that .zetarc file is not commited on your source control.                
   |                                                                                                                     
   | If you are using git, add .zetarc entry in .gitignore file  


I provide my password through the CLI

When you want to run your custom cloud services locally, you can provide your password using --developer-password option:

$ zeta run --developer-password "<your password>"


If you run the command through npm, you can provide the option like this:

$ npm start -- --developer-password "<your password>"


   | Sensitive information leak
/!\| Using this method writes your password in your terminal history. If your computer can be accessed by other people, they may be able  
   | to see your password in command history.                                                                                             


I provide my password in environment variable



TODO: https://github.com/zetapush/documentation/issues/19



  ‌
 
INFO    2018-07-13T14:20:32.612Z - COMMAND            | zetaPush(testing-projects/empty-app) -> [npm run deploy -- -vvv] 
ERROR   2018-07-13T14:20:33.548Z - COMMAND            | zetaPush(testing-projects/empty-app) -> [npm run deploy -- -vvv] {"exitCode":52}
INFO    2018-07-13T14:20:33.549Z - USER ACTION        | >>> User action DONE: zeta push from console 
ERROR   2018-07-13T14:20:33.547Z - PROCESS STDOUT/STD | 
[ERROR] Process is not run in a terminal (no TTY available). Can't prompt for developer password [ TypeError: process.stdin.setRawMode is not a function
    at prompt (/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/cli/node_modules/prompt-sync/index.js:68:34)
    at getDeveloperPassword (/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/cli/src/utils/security.js:26:12)
    at load (/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/cli/src/loader/config.js:118:34)
    at <anonymous> ] ‌
[ERROR] Missing required information  ‌
[ERROR] Push failed [ { code: 'CONFIG_MISSING_REQUIRED_INFO',
    message: 'Missing required information',
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'user@zetapush.com',
       developerPassword: '',
       workerServiceId: undefined } } ] ‌
npm ERR! code ELIFECYCLE
npm ERR! errno 52
npm ERR! empty-project@0.0.1 deploy: `zeta push "-vvv"`
npm ERR! Exit status 52
npm ERR! 
npm ERR! Failed at the empty-project@0.0.1 deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/aurelien/.npm/_logs/2018-07-13T14_20_33_540Z-debug.log
 
DEBUG   2018-07-13T14:20:33.558Z - GIVEN              | >> Apply Given 
INFO    2018-07-13T14:20:33.560Z - COMMAND            | npmInstallLatestVersion(testing-projects/empty-app) 
INFO    2018-07-13T14:20:33.558Z - GIVEN              | >>> Given testing application: testing-projects/empty-app 
SILLY   2018-07-13T14:20:33.559Z - GIVEN              | GivenTestingApp:execute -> npmInstallLatestVersion(testing-projects/empty-app) 
SILLY   2018-07-13T14:20:33.930Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] 
SILLY   2018-07-13T14:20:38.351Z - PROCESS STDOUT/STD | 

> get-cursor-position@1.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position
> node-gyp rebuild

make : on entre dans le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »
  CXX(target) Release/obj.target/pos/src/pos.o
  SOLINK_MODULE(target) Release/obj.target/pos.node
  COPY Release/pos.node
make : on quitte le répertoire « /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/get-cursor-position/build »

> bufferutil@3.0.5 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/bufferutil
> prebuild-install || node-gyp rebuild


> utf-8-validate@4.0.2 install /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/node_modules/utf-8-validate
> prebuild-install || node-gyp rebuild

+ @zetapush/cli@0.30.0-alpha.743bc6df
added 208 packages in 4.062s 
SILLY   2018-07-13T14:20:38.350Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/cli@canary --save] ->  
SILLY   2018-07-13T14:20:38.354Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] 
SILLY   2018-07-13T14:20:40.201Z - COMMAND            | npmInstallLatestVersion() -> [npm install @zetapush/platform@canary --save] ->  
SILLY   2018-07-13T14:20:40.207Z - COMMAND            | symlinkLocalDependencies(testing-projects/empty-app) 
WARN    2018-07-13T14:20:38.353Z - PROCESS STDOUT/STD | 
../src/pos.cc: In function ‘void Method(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/pos.cc:243:6: warning: variable ‘ret’ set but not used [-Wunused-but-set-variable]
  int ret, row, col;
      ^ 
SILLY   2018-07-13T14:20:40.203Z - PROCESS STDOUT/STD | 
+ @zetapush/platform@0.30.0-alpha.743bc6df
updated 1 package in 1.484s 
WARN    2018-07-13T14:20:40.206Z - PROCESS STDOUT/STD | 
 
SILLY   2018-07-13T14:20:40.607Z - GIVEN              | GivenTestingApp:execute -> setAccountToZetarc(testing-projects/empty-app, {"developerLogin":"user@zetapush.com","developerPassword":""}) 
DEBUG   2018-07-13T14:20:40.609Z - COMMAND            | readZetarc(testing-projects/empty-app) 
DEBUG   2018-07-13T14:20:40.611Z - COMMAND            | readZetarc(testing-projects/empty-app) ->  
INFO    2018-07-13T14:20:40.612Z - USER ACTION        | >>> User action: zeta run from console 
ERROR   2018-07-13T14:20:41.650Z - PROCESS STDOUT/STD | 

> empty-project@0.0.1 start /mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app
> zeta run "-vvv"

[LOG] Load and merge configuration with this priority order:
      1) command line arguments
      2) environment variables
      3) from file defined in ./.zetarc  ‌
[TRACE] Try to load conf from cli provess.argv  ‌
[TRACE] Try to load conf from process.env  ‌
[TRACE] Try to load conf from filesystem [ '.' ] ‌
[TRACE] Configuration loaded from file [ '.',
  { config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       developerLogin: 'user@zetapush.com' },
    filepath: '/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/integration/testing-projects/empty-app/.zetarc' } ] ‌
[TRACE] Using default values  ‌
[TRACE] merged config [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'user@zetapush.com',
    developerPassword: undefined,
    workerServiceId: undefined } ] ‌
[TRACE] config is not valid [ { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
    appName: undefined,
    developerLogin: 'user@zetapush.com',
    developerPassword: '',
    workerServiceId: undefined } ] ‌
[TRACE] not config issue  ‌
[TRACE] not a network error  ‌
[TRACE] access denied [ { code: 'CONFIG_MISSING_REQUIRED_INFO',
    message: 'Missing required information',
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'user@zetapush.com',
       developerPassword: '',
       workerServiceId: undefined } } ] ‌
[TRACE] no developer password  ‌
[TRACE] Analyze done => errorCode=ACCOUNT-02. Displaying help for this code  ‌
[TRACE] Display help message for ACCOUNT-02  ‌
[TRACE] Cache expired=false  ‌
[TRACE] Use cached troubleshoot for ACCOUNT-02  ‌
(?) 
You did provide your ZetaPush account login but not your password.


I provide my password in .zetarc file

Edit the .zetarc file to add developerPassword with your password:

{                                         
  "developerLogin": "<your login>",       
  "developerPassword": "<your password>"  
}                                        

   | Sensitive information leak
/!\| You must check that .zetarc file is not commited on your source control.                
   |                                                                                                                     
   | If you are using git, add .zetarc entry in .gitignore file  


I provide my password through the CLI

When you want to run your custom cloud services locally, you can provide your password using --developer-password option:

$ zeta run --developer-password "<your password>"


If you run the command through npm, you can provide the option like this:

$ npm start -- --developer-password "<your password>"


   | Sensitive information leak
/!\| Using this method writes your password in your terminal history. If your computer can be accessed by other people, they may be able  
   | to see your password in command history.                                                                                             


I provide my password in environment variable



TODO: https://github.com/zetapush/documentation/issues/19



  ‌
 
INFO    2018-07-13T14:20:40.612Z - COMMAND            | zetaRun(testing-projects/empty-app) -> [npm run start -- -vvv] 
ERROR   2018-07-13T14:20:41.651Z - COMMAND            | zetaRun(testing-projects/empty-app) -> [npm run start -- -vvv] {"exitCode":52}
INFO    2018-07-13T14:20:41.651Z - USER ACTION        | >>> User action DONE: zeta run from console 
ERROR   2018-07-13T14:20:41.650Z - PROCESS STDOUT/STD | 
[ERROR] Process is not run in a terminal (no TTY available). Can't prompt for developer password [ TypeError: process.stdin.setRawMode is not a function
    at prompt (/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/cli/node_modules/prompt-sync/index.js:68:34)
    at getDeveloperPassword (/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/cli/src/utils/security.js:26:12)
    at load (/mnt/data/developpement/zetapush-celtia/zetapush-js-sdk/packages/cli/src/loader/config.js:118:34)
    at <anonymous> ] ‌
[ERROR] Missing required information  ‌
[ERROR] Run failed [ { code: 'CONFIG_MISSING_REQUIRED_INFO',
    message: 'Missing required information',
    config: 
     { platformUrl: 'https://celtia.zetapush.com/zbo/pub/business',
       appName: undefined,
       developerLogin: 'user@zetapush.com',
       developerPassword: '',
       workerServiceId: undefined } } ] ‌
npm ERR! code ELIFECYCLE
npm ERR! errno 52
npm ERR! empty-project@0.0.1 start: `zeta run "-vvv"`
npm ERR! Exit status 52
npm ERR! 
npm ERR! Failed at the empty-project@0.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/aurelien/.npm/_logs/2018-07-13T14_20_41_639Z-debug.log
 
INFO    2018-07-13T14:20:41.723Z - USER ACTION        | >>> User action: 1) npm init from console 
INFO    2018-07-13T14:20:41.724Z - COMMAND            | npmInit(aurelien.baudet+celtia@zetapush.com, password, project-nominal-case) 
